images/ceph/Dockerfile

```diff
@@ -26,5 +26,9 @@ COPY rook rookflex toolbox.sh set-ceph-debug-level /usr/local/bin/
 COPY ceph-monitoring /etc/ceph-monitoring
 COPY rook-external /etc/rook-external/
 COPY ceph-csv-templates /etc/ceph-csv-templates
+RUN useradd rook -u 2016 # 2016 is the UID of the rook user and also the year of the first commit in the project
```

Comment:
The operator mounts an emptydir to `/var/lib/rook`, so that would overwrite what's already in the container. So do we need an init container like the ceph daemons? Or can we remove that emptydir now? The emptydir was required on openshift, so we should test there in any case.

Reply:
Let's discuss I'm not clear of the role of that emptydir anymore.

Reply:
Is it because the root fs is read-only on openshift?

Reply:
> Is it because the root fs is read-only on openshift?

The SCC sets RootReadOnly to false. I'm not sure, maybe?